src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/generate.cmd

```diff
@@ -8,7 +8,7 @@ setlocal
 
 set autoRestVersion=0.17.0-Nightly20160824
```

Comment:
Then please ignore this cmd file and we are going to follow the new instruction generate code locally.